### PR TITLE
[ICDS Dashboard] Add Django view for the API

### PR DESCRIPTION
**Pull request by Rohit25negi**
This code adds the django view for the Bihar API.
the variable change from `GOVERNANCE_API_PAGE_SIZE` to `STANDARD_API_PAGE_SIZE` is just for code simplification and reusability.

### CODE REVIEW

**Comment by Rohit25negi** [comment]
@calellowitz  this Pr along with other Bihar API PRs(https://github.com/dimagi/commcare-hq/pull/26944, https://github.com/dimagi/commcare-hq/pull/26929) is quite urgent. I am planning to deploy them on Thursday night IST so that we have some time to test as well. Please review. I am happy to be available on EST time for some hours if that makes the review faster

File: custom/icds_reports/migrations/sql_templates/database_views/bihar_demographics.sql
Comment (Rohit25negi) [inline custom/icds_reports/migrations/sql_templates/database_views/bihar_demographics.sql]: took from `bihar_demographics` table so that filter can use the index created on that table.

File: custom/icds_reports/const.py
Comment (calellowitz) [inline custom/icds_reports/const.py]: Nit: may be good to flag that this is cas, like `CAS_API_PAGE_SIZE`, since its definitely possible other ones would be different, like in our core data APIs

File: custom/icds_reports/migrations/sql_templates/database_views/bihar_demographics.sql
Comment (calellowitz) [inline custom/icds_reports/migrations/sql_templates/database_views/bihar_demographics.sql]: Should there be spaces here between the wards?

File: custom/icds_reports/reports/bihar_api.py
Comment (calellowitz) [inline custom/icds_reports/reports/bihar_api.py]: print statement

File: custom/icds_reports/urls.py
Comment (calellowitz) [inline custom/icds_reports/urls.py]: Not sure if this is committed yet, but it might be worth adding prepending something for all the bihar APIs at the front like we do with governance ones. so it would be `bihar_demographics/household_member_data/` or similar so that all the bihar ones share a path and if we have other household ones it is clear what we mean.

File: custom/icds_reports/views.py
Comment (calellowitz) [inline custom/icds_reports/views.py]: Might be nice to call this BiharDemographicsAPI, to be clear here (and because that phrase is used elsewhere)

Comment (calellowitz) [reply custom/icds_reports/views.py]: We have a bunch of very similar API views (and data fethcing functions), is it possible to share any of this functionality between them. It feels like duplicate a bunch every time

File: custom/icds_reports/reports/bihar_api.py
Comment (calellowitz) [inline custom/icds_reports/reports/bihar_api.py]: does this need to be a list? or can you use the queryset directly in the view

**Review by calellowitz — APPROVED** [review]
Lots of small comments, nothing blocking, but would be good to fix them before merging if you can

**Comment by czue** [comment]
Looks like Cal already did a solid review so I'll defer to him

File: custom/icds_reports/migrations/sql_templates/database_views/bihar_demographics.sql
Comment (Rohit25negi) [reply custom/icds_reports/migrations/sql_templates/database_views/bihar_demographics.sql]: yep. right. thanks

File: custom/icds_reports/urls.py
Comment (Rohit25negi) [reply custom/icds_reports/urls.py]: I can check with Nitish.

File: custom/icds_reports/const.py
Comment (Rohit25negi) [reply custom/icds_reports/const.py]: Done

File: custom/icds_reports/reports/bihar_api.py
Comment (Rohit25negi) [reply custom/icds_reports/reports/bihar_api.py]: Done

File: custom/icds_reports/views.py
Comment (Rohit25negi) [reply custom/icds_reports/views.py]: yep. created one base class and used for this view and for Disha API for now

**Review by calellowitz — APPROVED** [review]
Thanks for the followup

File: corehq/toggles.py
Comment (orangejenny) [inline corehq/toggles.py]: For future PRs, it's helpful to stick new flags in the middle of the file - we get merge conflicts when everybody is adding flags to the end of the file.

**Pull request merged**
